.eslintrc

```diff
@@ -1,3 +1,12 @@
 {
-    "extends": "eslint:recommended"
```

Comment:
I would prefer to mirror the existing rules we have. I want to keep changes like this as tight as possible.  the choice of tool shouldn't force us to touch every JS file in the project to match an new and arbitrary set of rules.  
I want to discuss these rules. I, for example, don't like trailing commas. I'm open to discuss everything on the project, but changing a bunch of stuff at once diffuses the opportunity for discussion.